server/controllers/utils.ts

```diff
@@ -378,54 +378,61 @@ createOneTeam = (teamSize: number, numRounds: number, people: any[]) => {
   return rounds;
 };
 
-export const createDynamicTeams = (teamSize: number, numRounds: number, dynamicOptions: boolean) => {
+export const createDynamicTeams = (teamSize: number, numRounds: number) => {
   /*
   * Returns array of {roundPairs, roundGen}
   * e.g.:
   * {roundPairs: [[1,6], [2,3], [4,5]], roundGen: {<some roundGen with 50% - n, 50% - 1 teamSize structure>}}
   * first round in pair is always with 1 user in a team*/
   const availableNumbers = Array.from(Array(numRounds).keys());
-  const roundPairs = consecutivePairs(availableNumbers);
-  console.log("Round pairs: " + roundPairs)
+  const roundPairs = randomPairs(availableNumbers); // pairs themselves are random
+
+  console.log("Confirming round pairs:")
+  for(var i = 0; i < roundPairs.length; i++){
+    console.log("pair: " + roundPairs[i])
+  }
```

Comment:
I have a for loop allergy. 
```suggestion
  roundPairs.forEach(p => console.log(`pair: ${p}`))
```

Though because you already said its 'confirming round pairs`, you could just do:
```javascript
roundPairs.forEach(console.log)
```
Which will just print each item on a new line.